Sources/_Datadog_Private/ObjcOSActivityUtils.m

```diff
@@ -0,0 +1,18 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+#import <Foundation/Foundation.h>
+#import "ObjcOSActivityUtils.h"
+
+@implementation ObjcOSActivityUtils
+
+static os_activity_t _currentActivity = OS_ACTIVITY_CURRENT;
```

Comment:
if the host app starts another activity (eg: `HostAppCustomButtonActivity`), does it affect us?
(i don't think this is gonna happen, i'm just curious)

Reply:
Yes, its true can be returned directly, i just followed the class property standard implementation, but better returning directly.

If the app starts another activity, depending how it is created we could fail to find current Span while in the activity. We can locate a current span in the current activity or in its parent activity, but we cannot iterate more